src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -570,8 +572,12 @@ public DevUtil(File buildDirectory, File serverDirectory, File sourceDirectory,
     }
 
     private void initGenerationContext() {
-        this.genContextDir = generateToSrc ? configDirectory : serverDirectory;
-        this.generatedFeaturesFile = new File(genContextDir, BinaryScannerUtil.GENERATED_FEATURES_FILE_PATH);
+        this.generatedFeaturesFileParent = generateToSrc ? configDirectory : generatedFeaturesTmpDir;
+        this.generatedFeaturesFile = new File(generatedFeaturesFileParent, BinaryScannerUtil.GENERATED_FEATURES_FILE_PATH);
```

Comment:
It probably doesn't have a negative effect... but I'm just curious if there is a particular reason that the generated file for the temp dir has to be at "configDropins/overrides" within the temp dir?

Reply:
No particular reason except to be careful. Sometimes we pass a path around instead of a base file name. I'm not sure if this would have an effect but I wanted to keep the two ways of handling the file similar.